src/reports/npe01__Nonprofit_Edition_Reports/NPSP_Campaign_Household_Mailing_List.report

```diff
@@ -36,7 +36,8 @@
     <columns>
         <field>Account.Informal_Greeting__c</field>
     </columns>
-    <description>Lists addresses for a mailing campaign, avoiding household duplicates - RUN FROM BUTTON IN CAMPAIGN</description>
+    <description>REPORT DEPRECATED 2017 NOVEMBER.
```

Comment:
Can we say something like: NO LONGER USED with Household Mailing List button on Campaign. Replaced with <new report name>.